app/_src/policies/meshtimeout.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Mesh Timeout (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [Timeout policy](../timeout).
+{% endwarning %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+| ----------------- | --------- | --- | ---- |
+| Mesh              | ✅         | ✅   | ✅    |
+| MeshSubset        | ✅         | ❌   | ❌    |
+| MeshService       | ✅         | ✅   | ❌    |
+| MeshServiceSubset | ✅         | ❌   | ❌    |
+| MeshGatewayRoute  | ❌         | ❌   | ❌    |
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+This policy enables {{site.mesh_product_name}} to set timeouts on the inbound and outbound connections depending on the protocol. Using this policy you can configure TCP and HTTP timeouts. Timeout configuration is split into two sections: common configuration and http configuration. Common config is applied to both HTTP and TCP communication.
+
+MeshTimeout policy lets you configure multiple timeouts:
+
+- connectionTimeout
+- idleTimeout
+- http requestTimeout
+- http streamIdleTimeout
+- http maxStreamDuration
+- http maxConnectionDuration
+
+### Timeouts explained
+
+#### Connection timeout
+
+Connection timeout specifies the amount of time DP will wait for a TCP connection to be established.
+
+#### Idle timeout
+
+Idle timeout is the amount of time that the DP will allow a connection to exist with no inbound or outbound activity.
```

Comment:
I think the explanation for this timeout should be different depending on the protocol:
* TCP - idle timeout is the amount of time that the DP will allow a connection to exist with no inbound or outbound activity.
* HTTP - time at which a downstream or upstream connection will be terminated if there are no active streams

Reply:
changed